tools/clang/test/HLSLFileCheck/shader_targets/hull/static_res_in_patch_constant_func.hlsl

```diff
@@ -0,0 +1,39 @@
+// RUN: %dxc -E main -T hs_6_0  %s 2>&1 | FileCheck %s
+
+// Make sure generated buffer load for static buffer used in patch constant function.
+// CHECK:call %dx.types.ResRet.f32 @dx.op.bufferLoad.f32(i32 68,
+
+struct HSPerPatchData
+{
+	float	edges[3] : SV_TessFactor;
+	float	inside   : SV_InsideTessFactor;
+};
+
+Buffer<float> buf;
+
+const static Buffer<float> static_buf = buf;
```

Comment:
I think we should also have a test where the static global is modified in the hull shader function and verify that it is not modified in the patch constant function.